tests/test_modeling_utils.py

```diff
@@ -920,3 +922,19 @@ def test_ddpm_ddim_equality_batched(self):
 
         # the values aren't exactly equal, but the images look the same visually
         assert np.abs(ddpm_images - ddim_images).max() < 1e-1
+
+    @slow
+    def test_karras_ve_pipeline(self):
```

Comment:
Very cool!!! (Did you also try it with the 1024 one?)

Reply:
Yes, the results are pretty ok :) 
(Although they can be made better with a bit of grid search to find the optimal `s_churn`, `s_noise`, etc, since the paper was dealing with much smaller models)